site/_posts/2023-01-12-what-happens-after-authentication-event.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: blog-post
+title: What happens to the tokens after an OAuth Authorization Code grant?
+description: What should you do with tokens returned after an OAuth grant? How can they be used by your application to ensure that only the correct users get access to data and functionality?
+author: Dan Moore
+image: TBD
+category: article
+tags: oauth authentication jwt cookie explainer mobile
+excerpt_separator: "<!--more-->"
+---
+
+At the end of the OAuth Authorization Code grant, after a user has logged in, one or more tokens are presented. This happens when you make a request to the token endpoint. 
```

Comment:
"you" is fairly ambiguous at this point. It might make more sense to clarify who "you" is during this token exchange. Maybe change to "after a user presents their credentials at login a code is returned which can be exchanged for one or more tokens at the token endpoint"